GSA/searchgov-spider
# SRCH-5530 Job Storage

## Summary
This changes the finished job storage for scrapyd from memory to sqlite. 

To test:
* check out new code
* start scrapyd 
* run a job using a curl request to scrapyd (e.g. `curl http://localhost:6800/schedule.json -d project=search_gov_spiders -d spider=domain_spider -d allowed_domains=quotes.toscrape.com -d start_urls=https://quotes.toscrape.com/`) or using scrapydweb.
* Verify existence of jobs.db in `dbs_dir` as specified in `scrapyd.conf`, probably `search_gov_crawler/dbs/`
* Verify a record in output of running `sqlite3 <path_to_dbs_dir>/jobs.db 'select * from finished_jobs;` matches the job you ran earlier.
* Restart scrapyd and confirm jobs.db still exists and contains records from previous runs.

### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.

#### Functionality Checks





#### Process Checks
